CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,126 @@
+# Contributor Covenant Code of Conduct
```

Comment:
We already have a CODE_OF_CONDUCT.md at the standard location.
Please see https://github.com/aklivity/zilla/blob/develop/.github/CODE_OF_CONDUCT.md.